service/src/main/java/bio/terra/profile/app/controller/ProfileApiController.java

```diff
@@ -82,7 +82,7 @@ public ResponseEntity<SamPolicyModel> addProfilePolicyMember(
     AuthenticatedUserRequest user = authenticatedUserRequestFactory.from(request);
     SamPolicyModel policy =
         profileService.addProfilePolicyMember(id, policyName, requestBody.getEmail(), user);
-    return new ResponseEntity<>(policy, HttpStatus.OK);
+    return new ResponseEntity<>(policy, HttpStatus.CREATED);
```

Comment:
While writing the test, I noticed that Swagger advertised 201, but we were returning 200. I think 201 is more correct, so changing this to match Swagger.

I looked through the Rawls usages of this method and they don't check the status code. I am hoping this is a low risk change.